test/chemistry/test_app_mgse.py

```diff
@@ -29,6 +30,8 @@
 from qiskit.chemistry.core import QubitMappingType
 from qiskit.chemistry.drivers import PySCFDriver, UnitsType
 
+# TODO Ground state interface PR
+
```

Comment:
Now that the unittests of the new framework are in, all of these TODO labels should be removed.

Reply:
I left some TODO for the eom tests that need to be update after the excited states interfaces is there.